packages/api/public/spec/schemas/parameters.yml

```diff
@@ -0,0 +1,24 @@
+languageCode:
+  name: code
+  in: path
+  description: IETF code of the language
+  required: true
+  schema:
+    type: string
+  example: es
```

Comment:
I extracted the URL parameters to a common file since they are repeated in multiple URLs.